tests/e2e/cucumber/features/smoke/fullTextSearch.ocis.feature

```diff
@@ -91,15 +91,17 @@ Feature: Search
       | withTag.txt                   |
       | spaceFolder/spaceTextfile.txt |
     When "Brian" opens the following file in texteditor
-      | resource                      |
-      | textfile.txt                  |
+      | resource     |
+      | textfile.txt |
     And "Brian" closes the file viewer
-    Then following resources should be displayed in the files list for user "Brian"
-      | resource                      |
-      | textfile.txt                  |
-      | testFolder/innerTextfile.txt  |
-      | fileToShare.txt               |
-      | fileWithTag.txt               |
-      | withTag.txt                   |
-      | spaceFolder/spaceTextfile.txt |
+    # TODO: uncomment the following step after the bug is fixed
+    # issue: https://github.com/owncloud/web/issues/9428
+    # Then following resources should be displayed in the files list for user "Brian"
+    #   | resource                      |
+    #   | textfile.txt                  |
+    #   | testFolder/innerTextfile.txt  |
+    #   | fileToShare.txt               |
+    #   | fileWithTag.txt               |
+    #   | withTag.txt                   |
+    #   | spaceFolder/spaceTextfile.txt |
```

Comment:
Since, we have issue https://github.com/owncloud/web/issues/9428, this step fails so I have commented out this step. And should be uncommented after the issue is fixed.

CC @ScharfViktor